po/zh-CN.po

```diff
@@ -19387,7 +19387,7 @@ msgstr ""
 
 #: src/glossary.md:72
 msgid "mutable:"
-msgstr "可变"
+msgstr "可变:"
```

Comment:
This change is inconsistent with others, like line 19382 and 19377. I agree this need to be changed, but it is better to change all related in a bulk.

Reply:
@NoahDragon Yes, This commit is to resolve Poedit's warning, so you may wander why,  but all of it is suggestion of Poedit

<img width="804" alt="image" src="https://github.com/google/comprehensive-rust/assets/11788119/00276a00-b473-43cc-933f-69bce3a8df26">

Reply:
Thank you! I would recommend to separate the Poedit linter check warning fix into another PR. These warnings are not really useful for Chinese translation, and have many discussion already:
* https://stackoverflow.com/questions/44644152/how-to-hide-silly-warnings-in-poedit
* https://github.com/vslavik/poedit/issues/395
* https://github.com/vslavik/poedit/issues/445

@mgeisler Do you think if it is okay to ingore the linter warnings?

Reply:
Thanks @NoahDragon for looking this up! I would be very happy to have language-specific advice in STYLE or TRANSLATIONS or somewhere else — whatever feels most natural to the language communities.